conf/docker-dcm/rsalsrv.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM centos:8
```

Comment:
This might also be better left at centos:7 (server-side component hasn't been tested on cent8).

Reply:
Reverted. Thanks.